lib/driver.js

```diff
@@ -51,6 +51,13 @@ class AndroidDriver extends BaseDriver {
     this.chromedriver = null;
     this.apkStrings = {};
     this.bootstrapPort = opts.bootstrapPort || DEVICE_PORT;
+    this.unlocker = helpers.unlocker;
+
+    // Move this up to do constructor since we are going to need
```

Comment:
No, it's a comment because this was outside of the driver class itself. I can remove the comment I just leave it there for review.